.github/workflows/tf-vars.yml

```diff
@@ -27,27 +24,33 @@ jobs:
       - name: Download Terraform Cleaner
         run: |
           VERSION="${{ github.event.inputs.version }}"
-          wget "https://github.com/sylwit/terraform-cleaner/releases/download/v${VERSION}/terraform-cleaner_${VERSION}_linux_amd64.tar.gz"
-          tar -xvf "terraform-cleaner_${VERSION}_linux_amd64.tar.gz"
-
+          wget https://github.com/sylwit/terraform-cleaner/releases/download/v${{ github.event.inputs.version || '0.0.4' }}/terraform-cleaner_${{ github.event.inputs.version || '0.0.4' }}_linux_amd64.tar.gz
+          tar -xvf terraform-cleaner_${{ github.event.inputs.version || '0.0.4' }}_linux_amd64.tar.gz
+          
       - name: Run Terraform Cleaner
         id: terraform-cleaner
         run: |
           mv terraform-cleaner /usr/local/bin
-          terraform-cleaner
-          cleaner_output=$(terraform-cleaner 2>&1)
-          echo "::set-output name=cleaner-output::$cleaner_output"
-
-      - name: Get PR Number
-        id: pr-number
+          echo "output=$(echo $(terraform-cleaner))" >> $GITHUB_ENV
+          output="${output//'%'/'%25'}"
```

Comment:
Update the output block if not remove these

Reply:
output="${output//'%'/'%25'}"
          output="${output//$'\n'/'%0A'}"
          output="${output//$'\r'/'%0D'}"
          these output block you are talking about ...?